docs/modeling/physical_models.rst

```diff
@@ -74,18 +74,9 @@ function gives the bolometric flux using
 The :meth:`~astropy.modeling.physical_models.BlackBody.lambda_max` and
 :meth:`~astropy.modeling.physical_models.BlackBody.nu_max` member functions
 give the wavelength and frequency of the maximum for :math:`B_\lambda`
-and :math:`B_\nu`, respectively, calculated using `Wein's Law
+and :math:`B_\nu`, respectively, calculated using `Wien's Law
 <https://en.wikipedia.org/wiki/Wien%27s_displacement_law>`_.
 
-.. note::
-
-    Prior to v4.0, the ``BlackBody1D`` and the functions ``blackbody_nu`` and ``blackbody_lambda``
```

Comment:
@dhomeier , regarding your comment, would leaving this note in but updating it be sufficient? Or maybe even making this a sub-section and move the `deprecated-blackbody` tag here. What do you think?

Reply:
Yes, to clarify after reading up a bit on the discussion in #9434 and #9282, I was just thinking of a brief upgrade guide in that comment. Could perhaps be a sub-section as well, but the comment layout might make it clearer that it is an outdated interface.

Reply:
I added a commit to the doc that points to the deprecated stuff in a previous release. Does this look good to you?

Reply:
Ah, forgot that the LTS docs would also still be around! Yes, that's a good solution.